services/wallet/transfer/commands.go

```diff
@@ -240,18 +240,20 @@ func (c *transfersCommand) Run(ctx context.Context) (err error) {
 
 			c.processUnknownErc20CommunityTransactions(ctx, allTransfers)
 
-			err = c.processMultiTransactions(ctx, allTransfers)
-			if err != nil {
-				log.Error("processMultiTransactions error", "error", err)
-				return err
-			}
-
 			if len(allTransfers) > 0 {
+				// First, try to match to any pre-existing pending/multi-transaction
```

Comment:
we were getting duplicate entries before this (multi transaction created by the detector + the one triggered by the user)